pkg/install/install.go

```diff
@@ -28,7 +28,7 @@ var kourierVersion = "0.24.0"
 var eventingVersion = "0.24.1"
 
 // Kourier installs Kourier networking layer from Github YAML files
-func Kourier() error {
+func Kourier(local string) error {
```

Comment:
Im finding it hard to guess what the parameter does from the name. Maybe we could rename it and/or document it a bit.